src/document/document.ts

```diff
@@ -446,8 +485,17 @@ export class Document<T, P extends Indexable = Indexable> {
    */
   private isUpdating: boolean;
 
+  /**
+   * `docEvents` stores all events in the document for replaying
+   * (time-traveling feature) in Devtools. Later, external storage such as
+   * IndexedDB will be used.
+   */
+  public docEvents: Array<Array<DocEvent<P>>>;
```

Comment:
Same with the below comment.
https://github.com/yorkie-team/yorkie-js-sdk/pull/760/files#diff-73a0a53c4c6680b52c4576ae1e7406ebf3bcc191406bf73851da233f6777a6bfR161